src/vm-status.js

```diff
@@ -0,0 +1,55 @@
+export function canStart (state) {
+  return state && (state === 'down' || state === 'paused' || state === 'suspended')
+}
+
+export function canShutdown (state) {
+  return state && (state === 'up' ||
```

Comment:
nitpicking: when refactoring, we can replace this syntax by 
```
return ['up', 'migrating', ...].includes(state)
```

Reply:
I had originally done that, but then I figured it was less efficient then the set of === || operators.  But, to satisfy my curiosity, I put together a quick benchmark across 7 different styles: http://jsben.ch/mhdZY

Net result, setting up the array outside of the function seems to be the fastest and most readable.  I'll adjust that way.

Reply:
It's nice comparison. I believe in case of UI development readability and clarity is usually more important than performance.

Reply:
I updated to indexOf() instead of includes().  They're within tens of ms of each other, but indexOf() is supported in IE.

Reply:
Please follow Marek's [suggestion][1] and use `include()`. I find it the cleanest and easiest to read solution.

---

If I understand the performance benchmark (http://jsben.ch/mhdZY) correctly, processing of predefined arrays was faster only because the time to evaluate the setup block is excluded from the test. Hence inlining of array definition shouldn't cause a performance penalty.

[1]: https://github.com/oVirt/ovirt-web-ui/pull/581#discussion_r186947553